includes/wikia/services/MediaQueryService.class.php

```diff
@@ -13,6 +13,11 @@ class MediaQueryService extends WikiaService {
 	const SORT_RECENT_FIRST   = 'recent';
 	const SORT_POPULAR_FIRST  = 'popular';
 	const SORT_TRENDING_FIRST = 'trend';
+	/**
+	 * @var string SORT_TRENDING_FIRST_LEGACY
+	 * Legacy sorting option used by requests from Game Guides apps
+	 */
+	const SORT_TRENDING_FIRST_LEGACY = 'trending';
 
 	const DB_RECENT_COLUMN    = 'added_at';
 	const DB_POPULAR_COLUMN   = 'views_total';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "views_total" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)